src/snippets/ambient-setup.rst

```diff
@@ -1,8 +1,11 @@
-1. Activate a Conda environment that you wish to use:
+1. Create a new Conda environment and install Python 3.10.13:
 
    .. code-block:: bash
 
+      conda create -n <your-environment-name>
       conda activate <your-environment-name>
+      conda install --override-channels --strict-channel-priority \
+            -c conda-forge --yes python=3.10.13
```

Comment:
Is it necessary to split and pin `python` here? I'd assume the `conda install` command in the next step should auto-resolve to a compatible Python version.

I don't have `conda` on macOS so I can't test there immediately, but I checked on Linux via GitHub Codespace and it works fine without this extra step, auto-resolving Python to 3.8.19.

Reply:
I originally went down this rabbit hole because my initial install attempt _didn't_ resolve to an installable dep graph because Conda couldn't find a Python version that satisfied everybody. That said, at the point I was using the old version of Conda I had previously had installed, which didn't have the libmamba resolver stuff. 

I repeated the `conda` env build on macOS, without pining the Python version, and it works, and installs 3.8.15. 

Will remove the pining line.

Reply:
_Actually..._ you got me to look at this a little bit more, and it turns out `conda create` also supports all the `install` arguments, so I condensed this down a little bit more, as you'll see.

Reply:
This makes it less portable to users who want to install on top of an existing environment. But maybe it's fine to assume those people would know how to pull things out to craft their own `conda install` command. I'm cool with either.